### DISCUSSION TITLE
Update to German (de)

### DISCUSSION THREAD

**Discussion by MortenMacFly** [aclap-dev/video-downloadhelper#991] [Language & Translation]
[de.txt](https://github.com/user-attachments/files/16164860/de.txt)

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language 🙏

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
